launch/external_gps_demo.launch

```diff
@@ -0,0 +1,40 @@
+<!--
+this file provides an example of how to use an external gps ros message as input to the
+microstrain unit.
+
+this particular example was used to configure the system with a ublox RTK GPS which has
+substancially better precision than the internal microstrain GPS
+-->
+<launch>
+
+
+<!--
+Set up external gps source.  this example uses the ublox_gps node available at http://wiki.ros.org/ublox
+-->
+    <node pkg="ublox_gps" type="ublox_gps" name="rover_ublox"
```

Comment:
This example launch file is good but it makes this driver depend on `ublox_gps` as a run dependency.  My initial thought is not include it and the example live somewhere else because there are many different GPS units that could work with.  What are your thoughts